### ISSUE TITLE
`0.26 Beta`: TS 5.5 `@import` comments passthrough in declarations to documentation.

### ISSUE THREAD

**Issue by typhonrt** [TypeStrong/typedoc#2594]
Well.. TS 5.5 is almost out along with `@import` support. What I have noticed in testing is that TSC will retain the `@import` JSDoc comment blocks when generating declarations from ESM code. In the declarations the correct action `import` statements are added, but the JSDoc `@import` comment block is not removed.  This can cause minor documentation issues if the next symbol in the declarations doesn't have an associated comment block.. Say a class, or whatever comes next. Here is an example where `Test` doesn't have a leading comment block (`test.d.ts`) and thus TypeDoc will pick up the `@import` comment as associated with `Test`.

```ts
import { TestAlias } from 'test-package';

/**
 * @import { TestAlias } from 'test-package'
 */
declare class Test {}

/**
 * @type {TestAlias}
 */
declare const subTest: TestAlias;

export { subTest, Test };
```

TypeDoc will treat `@import` like any other unknown tag and output it:
![image](https://github.com/TypeStrong/typedoc/assets/311473/793ef4a5-7853-4769-b450-4f73c1c474c1)

--------------------

It's a relatively small issue, but one that I'm on the fence whether it is appropriate to post in the Typescript repo, so I'm seeking your opinion. I think all `@import` only comment blocks can be removed from generated declarations and maybe it's just an oversight from the TS team.

**Comment by Gerrit0** [comment]
Well that's annoying, I added code to make TypeDoc ignore `@import` comments, but I only added it for the file comment parser, I didn't consider the case you described here. This is a bug!

**Comment by Gerrit0** [comment]
Fixed with v0.26.0-beta.4
